gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/NewV4Api.java

```diff
@@ -15,42 +15,28 @@
  */
 package io.gravitee.apim.core.api.model;
 
-import io.gravitee.definition.model.DefinitionVersion;
-import io.gravitee.definition.model.v4.ApiType;
 import io.gravitee.definition.model.v4.analytics.Analytics;
 import io.gravitee.definition.model.v4.endpointgroup.EndpointGroup;
 import io.gravitee.definition.model.v4.failover.Failover;
 import io.gravitee.definition.model.v4.flow.Flow;
 import io.gravitee.definition.model.v4.flow.execution.FlowExecution;
 import io.gravitee.definition.model.v4.listener.Listener;
 import java.util.List;
-import java.util.Set;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.experimental.SuperBuilder;
 
+@EqualsAndHashCode(callSuper = true)
 @Data
-@Builder(toBuilder = true)
+@SuperBuilder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
 @Setter(lombok.AccessLevel.NONE)
-public class NewApi {
-
-    private String name;
-
-    private String apiVersion;
-
-    @Builder.Default
-    private DefinitionVersion definitionVersion = DefinitionVersion.V4;
-
-    private ApiType type;
-
-    private String description;
-
-    @Builder.Default
-    private Set<String> tags = Set.of();
+public class NewV4Api extends AbstractNewApi {
```

Comment:
Maybe use Http instead of just V4?
```suggestion
public class NewHttpApi extends AbstractNewApi {
```